internal/server/tasks_dashboard.go

```diff
@@ -39,9 +41,15 @@ func cardPayments(client CardPaymentsClient, tmpl Template) Handler {
 			return err
 		}
 
-		vars := cardPaymentsVars{
-			Tasks:             tasks,
-			XSRFToken:         ctx.XSRFToken,
+		vars := tasksDashboardVars{
+			Tasks:     tasks,
+			Title:     "Tasks Dashboard",
+			XSRFToken: ctx.XSRFToken,
+		}
+
+		if len(myDetails.Teams) > 0 {
+			teamName := strings.Trim(strings.ReplaceAll(myDetails.Teams[0].DisplayName, "Team", ""), " ")
+			vars.Title = teamName + " Dashboard"
```

Comment:
What would happen if a user didn't have a team? Is that being handled?

Reply:
If they've got no team, but they do have the "Self Allocation Task User" role, then they'll see the page with a generic "Tasks Dashboard" title (line 46) and when they request a task they'll get a 422 error (as we've just seen in production 😓)

Reply:
Cool as long as it's handled 😅